app/controllers/mentee_profiles_controller.rb

```diff
@@ -24,5 +50,12 @@ def missing_mentor
       redirect_to dashboard_mentee_profiles_path, notice: 'Couldnt send mail'
     end
   end
+  
+  private
+  def user_params
+    params.require(:user).permit(:first_name, :last_name, :country, :program_country, :timezone, :project_attributes => [:id, :title,:language,:description,:github_link, :mentor_id])
```

Comment:
This didnt work. It still updated the mentor_id equal to mentee_id, when u Submit.